beta/docker/compose.gw.yaml

```diff
@@ -0,0 +1,13 @@
+services:
+  scim:
+    secrets:
+      - workspace-credentials
+      - workspace-secrets
+    environment:
+      - OP_WORKSPACE_CREDENTIALS=/run/secrets/workspace-credentials.json
```

Comment:
Being set multiple places. I would lean using `scim.env` for all variables that end users will need to modify.

Reply:
Since end users will not be changing the location of workspace credentials, I retract this comment